config/locales/client.en.yml

```diff
@@ -584,6 +584,11 @@ en:
       date_filter: "Posted between"
       in_reply_to: "in reply to"
       filtered_flagged_by: "Flagged by"
+      unknown:
+        title: "You have pending plugin reviews of type:"
+        instruction: "They cannot be properly displayed until you enable the relevant plugin. Please enable the plugin and refresh the page. Alternatively, you can ignore them. <a href='%{url}' target='_blank'>More info</a>"
```

Comment:
"enable the relevant plugin" is a difficult instruction to follow, given that `Reviewable.type` isn't exposed anywhere else in the UI. It may be hard to guess which plugin is at fault here.

Perhaps explicitly stating that the "more info" link contains a list of known Reviewables and their corresponding plugin would help?

Reply:
> Perhaps explicitly stating that the "more info" link contains a list of known Reviewables and their corresponding plugin would help?

I am a bit conflicted about the consistency of "more info..." in other admin pages and making it more clear for admins. This warning is a bit of an edge case and I don't think many people will see it. I would like to merge it and see if that v0 is clear for admins. In the future probably we will register which plugin created Reviewable which will improve this warning anyway.